css/notes.css

```diff
@@ -138,20 +138,27 @@
 }
 
 #app-content .utils {
-    padding: 0 20px 0 20px;
-    line-height: 30px;
     position: fixed;
-    top: 45px;
-    right: 20px;
+    top: 58px;
+    right: 0;
```

Comment:
No, as some lines below there is padding added to the label. Did you actually try this out? ;)